app/assets/stylesheets/content/_work_packages_table.sass

```diff
@@ -129,6 +129,7 @@ table.workpackages-table
       &.checkbox
         min-width: 0
         width: 20px
+        padding: 7px 5px 3px 7px
```

Comment:
Another finding (I am fiddling in IE here, because of the alignment tools):

When I set the `line-height` to `1` **instead** of modifying the `padding`, this is the result:

![line-check](https://cloud.githubusercontent.com/assets/453584/8003052/65525d7a-0b74-11e5-8e8b-b97de1e9191c.png)

The alignment is roughly as good as your padding changes, but the row height does not increase at all.

However: When apply **both**, your padding changes and the line height changes, then the overall row height does not increase anymore, but the checkbox comes out a little too low. So probably one would want to apply both: An explicit line-height and some kind of padding... Dunno...